dataactcore/read_config.py

```diff
@@ -134,7 +124,7 @@
         category = CONFIG_CATEGORIES[category_yaml_name]
         category_message = "### {}".format(category_yaml_name)
         for key in CONFIG_LOGGING["values_to_log"][category_yaml_name]:
-            value = category[key]
+            value = category.get(key,"Value not provided in config")
```

Comment:
Can you clarify what behavior you'd like here?  Currently this is written to the log as "... , key: Value not provided in config, ..." where the ... is messages for other keys.  Would you like that written in a different way?